tests/flows/erc20_source_native_destination_multihop.go

```diff
@@ -16,6 +16,7 @@ import (
  * Deploy a ERC20 token source on the primary network
  * Deploys NativeDestination to Subnet A and Subnet B
  * Bridges C-Chain example ERC20 tokens to Subnet A as Subnet A's native token
+ * Bridges C-Chain example ERC20 tokens to Subnet B as Subnet B's native token to collateralize the bridge on Subnet B
```

Comment:
Should we add the latter part of this comment to the line above?

Reply:
I left it out because the tokens sent to Subnet A are used later in the multihop, whereas tokens are sent to Subnet B before the multihop portion of the test only to simplify the multihop logic.